app/src/main/kotlin/com/tailoredapps/androidapptemplate/detail/DetailView.kt

```diff
@@ -35,16 +37,16 @@ class DetailView(
      * This is just to show that injection in fragments work.
      * Please do not use a repo or service in the view.
      */
-    private val dataRepo: DataRepo
+    private val dataRepo: DataRepo,
 ) : Fragment(R.layout.fragment_detail) {
 
     private val binding by viewBinding(FragmentDetailBinding::bind)
     private val navController by lazy(::findNavController)
     private val viewModel by viewModel<DetailViewModel>()
 
-    init {
-        launchOnViewLifecycleStartCancelOnViewLifecycleStop {
-
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        repeatOnLifeCycleStart {
```

Comment:
```suggestion
        launchWhenStartedCancelWhenStopped {
```
Thats how we call it in our project.
The former name was bit long but I think it is nice to now it is cancelled on stop or what do you think?

If not we should add it to the kdoc of the extension when it is cancelled ("cancels when it falls below that state" is a bit _confusing_ at times)